bin/node/runtime/pangolin/src/pallets/bridge_ethereum_bsc.rs

```diff
@@ -0,0 +1,20 @@
+// --- substrate ---
+use sp_core::U256;
+// --- darwinia ---
+use crate::*;
+use darwinia_bridge_ethereum_bsc::{BSCConfiguration, Config};
+
+frame_support::parameter_types! {
+	pub Configuration: BSCConfiguration = BSCConfiguration {
```

Comment:
What cfg should be used here?

Reply:
https://github.com/darwinia-network/parity-bridges-common/blob/3fa0052c2d63b938b8c2a18283f206f0e47cdd83/modules/ethereum-bsc/src/mock.rs#L101

example here.
These parameters are the same as the prod environment I think, more like constants